convex/battle.ts

```
@@ -0,0 +1,1276 @@
+      }
+
+      case "point_calculation": {
+        // 自動的に次のラウンドへ
+        const updatedBattle = await ctx.db.get(battle_id);
+        if (updatedBattle && updatedBattle.game_status !== "finished") {
+          // 新しいお題カードを選択
+          const newFieldCard = await getRandomFieldCard(ctx);
+
+          // プレイヤー状態をリセット
+          const resetPlayers = updatedBattle.players.map((player) => ({
+            ...player,
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)